modifytags/modifytags.go

```diff
@@ -0,0 +1,367 @@
+package modifytags
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/token"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode"
+
+	"github.com/fatih/camelcase"
+	"github.com/fatih/structtag"
+)
+
+// A Mutation defines how the input is modified.
+type Mutation struct {
+	Remove        []string            // Remove tags for the comma separated list of keys.
+	RemoveOptions map[string][]string // Remove the list of options per given key.
+
+	Add                  []string            // Adds tags for the comma separated list of keys.
+	AddOptions           map[string][]string // Add the options per given key.
+	Override             bool                // Override current tags when adding tags.
+	SkipUnexportedFields bool                // Skip unexported fields
+
+	Transform   string // Adds a transform rule when adding tags. Current options: [snakecase, camelcase, lispcase, pascalcase, titlecase, keep].
+	Sort        bool   // Sort tags in increasing order according to the key name.
+	ValueFormat string // Format the given tag's value. i.e: \"column:{field}\", \"field_name={field}\"
```

Comment:
// format for the tag's value, after transformation; for example "column:{field}"